core/modules/widgets/text.js

```diff
@@ -31,7 +31,9 @@ TextNodeWidget.prototype.render = function(parent,nextSibling) {
 	this.computeAttributes();
 	this.execute();
 	var text = this.getAttribute("text",this.parseTreeNode.text || "");
+	var length = this.getAttribute("length");
```

Comment:
yea. but `this.getAttribute()` has 2 input parameters. The second one is the default value, that should be used. So the function call is already designed to return a valid param. 

undefined is a coincidence since `this.getAttirbute()` returns undefined atm. So whenever this behaviour may change in the future, the code will break. According to [Murphy's law](https://en.wikipedia.org/wiki/Murphy%27s_law) this code will break. But it could be easily avoided.